integration_service/migrations/0001_create_user_broker_credentials_table.py

```diff
@@ -0,0 +1,196 @@
+# Generated by Django 3.2 - Create User Broker Credentials Table
+
+from django.db import migrations, models
+import django.utils.timezone
+from django_mysql.models import EnumField
+from django.core.validators import MinLengthValidator
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        # No dependencies since this is foundational table
+    ]
+
+    operations = [
+        # Create UserBrokerCredential table
+        migrations.CreateModel(
+            name='UserBrokerCredential',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False)),
```

Comment:
What does serialise does here 
   ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False)), 
   Do we need to have this other model as well or drop this round this model as well?